### PR TITLE
add gangorra information

### PR DESCRIPTION
**Pull request by Gangorra**
Adding my information. Please tag hacktoberfest to participate in the event.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -53,6 +53,15 @@
       "github": "https://github.com/solankidivyam",
       "linkedin": "https://www.linkedin.com/in/divyam-solanki-7a9a12190/",
       "quote": "Designing makes me feel the happiness of the world..!"
+    },
+    {
+      "name": "Felipe Gangorra",
+      "batch": 2021,
+      "picture_url": "https://ibb.co/gV3PTh8",
```
Comment (nlok5923) [inline data.json:60]:
```suggestion
      "picture_url": "https://i.ibb.co/9tNVTS0/IMG-20210913-221751-399.jpg",
```

**Review by nlok5923 — CHANGES_REQUESTED** [review]
PTAL !!

**Review by Gangorra — COMMENTED** [review]
changed. I don't know if this is the right procedure, I'm still new to this

**Comment by Gangorra** [comment]
if you can help with tips i would appreciate it

**Comment by nlok5923** [comment]
@Gangorra I just suggested those changes. And it's not a good practice to commit those changes using web UI you should use CLI instead.

Anyways congratulations on making successful contribution 🎉 but do remember it next time.

**Review by nlok5923 — APPROVED** [review]
LGTM!!

**Comment by Gangorra** [comment]
So should I make the change to the downloaded repository and then do an updated push?

**Comment by nlok5923** [comment]
> So should I make the change to the downloaded repository and then do an updated push?

Yes

**Comment by Gangorra** [comment]
Thanks for your help

**Pull request merged**